build.gradle

```diff
@@ -1,71 +1,28 @@
 plugins {
     id "idea"
     id "groovy"
-    id "com.github.erdi.webdriver-binaries" version "2.2"
-    id "com.github.erdi.idea-base" version "2.2"
-}
-
-ext {
-    // The drivers we want to use
-    drivers = ["firefox", "chrome", "chromeHeadless"]
-
-    ext {
-        gebVersion = '5.1'
-        seleniumVersion = '3.14.0'
-        chromeDriverVersion = '79.0.3945.36'
-        geckoDriverVersion = '0.26.0'
-    }
 }
 
 repositories {
     mavenCentral()
 }
 
 dependencies {
-    // If using Spock, need to depend on geb-spock
-    testCompile "org.gebish:geb-spock:$gebVersion"
-
-    // If using JUnit, need to depend on geb-junit
-    testCompile "org.gebish:geb-junit4:$gebVersion"
-    testCompile "junit:junit:4.13.2"
-
-    // Drivers
-    testCompile "org.seleniumhq.selenium:selenium-chrome-driver:$seleniumVersion"
-    testCompile "org.seleniumhq.selenium:selenium-firefox-driver:$seleniumVersion"
-}
-
-webdriverBinaries {
-    chromedriver {
-        version = chromeDriverVersion
-        fallbackTo32Bit = true
-    }
-    geckodriver geckoDriverVersion
-}
-
-drivers.each { driver ->
-    task "${driver}Test"(type: Test) {
-        group JavaBasePlugin.VERIFICATION_GROUP
-
-        outputs.upToDateWhen { false }  // Always run tests
-
-        systemProperty "geb.build.reportsDir", reporting.file("geb/$name")
-        systemProperty "geb.env", driver
-    }
+    testImplementation 'io.github.bonigarcia:selenium-jupiter:4.0.1'
+    testImplementation 'org.seleniumhq.selenium:selenium-java:4.1.2'
+    testImplementation 'org.junit.jupiter:junit-jupiter-engine:5.8.1'
```

Comment:
This does change to project over to junit5. The readme needs to be updated as well.